README.md

```diff
@@ -4,18 +4,628 @@
 
 _visit www.actionherojs.com for more information_
 
-This tutorial showcases a realistic Actionhero project. In addition to the `actionhero` project, it uses the [`ah-seqeulzie-plugin`](https://github.com/actionhero/ah-sequelize-plugin) to use models and migrations against a Postgres database.
+This tutorial showcases a realistic Todo List Application with CRUD operations.
 
-## To install:
+To simply play around with the application, follow the next steps:
 
-You will need node.js, npm, redis, and postgres available.
+1. Make sure you've the following installed on your system
+    - Redis
+    - Postgresql 
+    - NodeJS
 
-`npm install`
+2. Clone the project
 
-## To Run:
+```
+git clone https://github.com/actionhero/actionhero-tutorial.git 
+&& cd actionhero-tutorial
+```
 
-`npm start`
+3. Install the dependencies
+
+```
+npm install
+```
+
+4. Configure the database connection in `./src/config/sequelize.js` and run migrations.
+```
+npx sequelize-cli db:migrate
```

Comment:
`automigrate` is enabled in config/sequelize.ts, so this isn't needed

Reply:
But doesn't it require a restart of dev server? 
That is why I added it, Will remove the whole sequelize-cli thing, it was conflicting anyways.

Reply:
I've left it there but added the comment about auto-migration feature available in AH.
Let me know if it works for you otherwise I'll remove it.